firefly/kernel/intel64/gdt/tss.cpp

```diff
@@ -8,10 +10,11 @@ inline void load_tss(SegmentSelector selector) {
 }
 
 void init(uint64_t stack) {
-    Tss& tss = thisCpu().tss;
-    gdt::setTssEntry(thisCpu().gdt, reinterpret_cast<uint64_t>(&tss), 0x20, 0x89);
-	tss.RSP0 = stack;
-    tss.IST1 = 0; // TODO: Implement me!
+    auto cpuData = static_cast<CpuData*>(getCpuData()->selfPointer);
+    Tss& tss = cpuData->tss;
+    gdt::setTssEntry(cpuData->gdt, reinterpret_cast<uint64_t>(&tss), 0x20, 0x89);
+    tss.RSP0 = stack;
+    tss.IST1 = 0;  // TODO: Implement me!
```

Comment:
<img src="https://media.vlipsy.com/vlips/nLW9ihzL/preview-small.jpg" />

Reply:
(IST)

Reply:
Forgot about this, I'll look at it